components/NewsAndUpdatesModal.js

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Query } from '@apollo/client/react/components';
+import { useRouter } from 'next/router';
+import { FormattedDate, FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { fontSize, margin } from 'styled-system';
+
+import { API_V2_CONTEXT, gqlV2 } from '../lib/graphql/helpers';
+
+import NextIllustration from './home/HomeNextIllustration';
+import Container from './Container';
+import { Flex } from './Grid';
+import HTMLContent from './HTMLContent';
+import StyledButton from './StyledButton';
+import StyledCarousel from './StyledCarousel';
+import StyledLink from './StyledLink';
+import Modal, { ModalBody, ModalFooter, ModalHeader } from './StyledModal';
+import { Span } from './Text';
+
+const Text = styled.p`
+  font-family: Inter;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 28px;
+  line-height: 36px;
+  color: #141414;
+  margin: 0px 16px;
+  ${margin}
+  ${fontSize}
+`;
+
+const PublishedDate = styled(Container)`
+  font-family: Inter;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 13px;
+  line-height: 20px;
+  color: #4e5052;
+`;
+
+const newsAndUpdatesQuery = gqlV2/* GraphQL */ `
+  query Update($collectiveSlug: String, $onlyChangelogUpdates: Boolean) {
+    account(slug: $collectiveSlug) {
+      updates(onlyChangelogUpdates: $onlyChangelogUpdates) {
+        nodes {
+          id
+          slug
+          publishedAt
+          title
+          summary
+        }
+      }
+    }
+  }
+`;
+
+const goToUpdatePage = (router, url, event) => {
+  event.preventDefault();
+  return router.push(url);
+};
+
+const NewsAndUpdatesModal = ({ onClose, ...modalProps }) => {
+  const router = useRouter();
+
+  return (
+    <Modal onClose={onClose} width="576px" {...modalProps}>
+      <ModalHeader>
+        <Flex width="100%">
+          <Flex>
+            <Span pt={1}>
+              <NextIllustration
+                width={41}
+                height={28}
+                src="/static/images/updates-and-news-modal-icon.svg"
+                alt="Updates and News Icon"
+              />
+            </Span>
+            <Text fontSize={['25px', '28px']}>News and Updates</Text>
+          </Flex>
+        </Flex>
+      </ModalHeader>
+      <hr />
+      <ModalBody>
+        <Query
```

Comment:
:ice_cream: No need to change it, but we tend to prefer using hooks (`useQuery`/`useMutation`) for new developments

Reply:
Thanks, I'll keep that in mind for future. 😄